environment-setup.md

```diff
@@ -0,0 +1,152 @@
+# 💻 Setting up your development environment
+
+There are many guides to setting up your computer for development. Each has its own merits. At DataMade, we perform a mix of maintaining legacy projects and developing new projects, often in parallel. Our toolkit is therefore optimized for managing many, isolated versions of packages and getting up and running on new projects quickly.
+
+#### Contents
+
+- [Version control](#version-control)
+- [Text editor](#text-editor)
+- [Python](#python)
+- [Docker](#docker)
+- [Security](#security)
+- [Data](#data)
+- [Geospatial data](#geospatial-data)
+- [Static sites](#static-sites)
+
+For Linux users, most of these tools are available as aptitude packages. When possible, prefer binary packages to building from source. For Mac users, most of these tools are available as [Homebrew](http://brew.sh/) packages. Homebrew is by far the easiest way to manage packages on OS X.
+
+## Version control
+
+We use GitHub and Git to keep our work under version control. Note that we prefer the `git` CLI to the GitHub desktop GUI.
```

Comment:
It might be helpful to note that git is usually built-in:

```suggestion
We use GitHub and Git to keep our work under version control. Note that we prefer the `git` CLI to the GitHub desktop GUI. The Git CLI is built-in on MacOS and most Linux distributions, usually with an acceptably recent version; to check whether you already have Git installed, run `which git`.
```